### PR TITLE
Devx 2849 voyagers

### PR DESCRIPTION
**Pull request by fabianrbz**
## Description

Please describe what the changes are made in this pull request and why the change was necessary.

## Deploy Notes

Notes regarding deployment the contained body of work. These should note any db migrations, etc.

### CODE REVIEW

File: app/views/static/default_landing/partials/_headshot.html.erb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<div class="Nxd-profile Vlt-center">
```
Comment (benhayehudi) [inline app/views/static/default_landing/partials/_headshot.html.erb:1]: Might be a good idea to add some `raise` arguments here for required parameters that are missing. For example:

```
<% 
    raise "missing 'name' key in headshot landing page block" unless local_assigns['name']
    raise "missing 'intro' key in headshow landing page block" unless local_assigns['intro']
%>
```

We do that throughout the partials to help guide people developing new landing pages work through errors.

Comment (fabianrbz) [reply app/views/static/default_landing/partials/_headshot.html.erb:1]: yeah, we are trying to define which values are required

**Comment by phazonoverload** [comment]
Sorry, my error. Thought that button did something else

**Comment by benhayehudi** [comment]
Is this ready for review again?

**Pull request merged**